tests/integration/bundles/kfp_1.7_stable_install.yaml.j2

```diff
@@ -24,10 +24,9 @@ applications:
     trust: true
   kfp-db:
     charm: mysql-k8s
-    channel: 8.0/edge
+    channel: 8.0/stable
     scale: 1
-    options:
-      profile: testing
```

Comment:
This option was suggested by the data team to have a constraint mysql.

Also, having `constraints: mem=250M` will just result in mysql not being able to even deploy.

Reply:
Yes. I put it back in.